src/components/NewLocationPage/AboveTheFold/AboveTheFold.tsx

```diff
@@ -49,8 +50,10 @@ const AboveTheFold: React.FC<AboveTheFoldProps> = React.memo(
     onClickSparkLine,
     onClickMasksCard,
     onClickTransmissionMetricsCard,
+    yPosition,
   }) => {
     const showMasksCard = locationSummary.level === Level.HIGH;
+    const offsetMap = yPosition ? yPosition > 5350 : false;
```

Comment:
This `5530` looks very fragile.  Any time we add/remove content on the location page it'll change this number, right?  Can we make it relative to the bottom of the page instead, so the number is only dependent on the footer size?  I'd also then put this into a constant called FOOTER_HEIGHT or something.